README.md

```diff
@@ -3,39 +3,65 @@
 ## Installation
 1. Clone repository into workspace [It doesn't have to be called moveit_simple_ws]
 ```
-mkdir moveit_simple_ws && cd moveit_simple_ws
-mkdir src && cd src
-catkin_init_workspace
+mkdir -p moveit_simple_ws/src && cd moveit_simple_ws/src
 git clone -b kinetic-devel https://github.com/plusone-robotics/moveit_simple.git
 ```
 
-2. Source Dependencies
+2. Get Source Dependencies
 ```
+cd moveit_simple_ws/src
 wstool init . moveit_simple/.travis.rosinstall
-cd ..
 ```
 
-3. Package Dependencies
+3. Install Package Dependencies
 ```
+cd moveit_simple_ws
 rosdep install --from-paths src --ignore-src --rosdistro $ROS_DISTRO -y
 ```
 
 4. Build
 ```
-catkin build --cmake-args -DCMAKE_BUILD_TYPE=Debug
-or
 catkin build --cmake-args -DCMAKE_BUILD_TYPE=Release
+```
+**NOTE:** To use gdb with this project set `-DCMAKE_BUILD_TYPE=Debug` instead.
 
+5. Source the Workspace
+```
+cd moveit_simple_ws
 source devel/setup.bash
 ```
 
-## Running Tests with Rviz Visualization
+**NOTE:** You will need to re-run step #5 in any new terminal
+
+
+## Building and Running Tests
+
+1.  Run the MoveIt Simple Tests
+```
+roscd moveit_simple
+catkin run_tests --no-deps --this -i
+```
+
+1. Get test results:
+```
+cd moveit_simple_ws
+catkin_test_results build/moveit_simple
+```
+
+### Run the Tests Using RViz as a Visualizer
+1. Launch the RViz visualizer
 ```
 roslaunch moveit_simple test_display.launch
+```
+
+2. Launch a Test. (Be sure to substitute `<Test Name>` with either `motoman_mh5` or `kuka_kr210`)
+```
 rostest moveit_simple <Test Name>_utest.launch -r
```

Comment:
Maybe obvious, but since we're touching this file already, it would be good to explicitly state that this instruction happens in a new terminal session.